packages/@ember/-internals/glimmer/lib/modifiers/custom.ts

```diff
@@ -109,18 +126,36 @@ class InteractiveCustomModifierManager<ModifierInstance>
     return new CustomModifierState(element, definition.delegate, instance, capturedArgs);
   }
 
-  getTag({ args }: CustomModifierState<ModifierInstance>): Tag {
-    return args.tag;
+  getTag({ args, tag }: CustomModifierState<ModifierInstance>): Tag {
+    return combine([tag, args.tag]);
   }
 
   install(state: CustomModifierState<ModifierInstance>) {
-    let { element, args, delegate, modifier } = state;
-    delegate.installModifier(modifier, element, args.value());
+    let { element, args, delegate, modifier, tag } = state;
+
+    let tracked = track(() => {
+      delegate.installModifier(modifier, element, args.value());
+    });
+
+    let { capabilities } = delegate;
+
+    if (capabilities === undefined || capabilities.disableLifecycleTracking !== true) {
+      update(tag, tracked);
+    }
   }
 
   update(state: CustomModifierState<ModifierInstance>) {
-    let { args, delegate, modifier } = state;
-    delegate.updateModifier(modifier, args.value());
+    let { args, delegate, modifier, tag } = state;
+
+    let tracked = track(() => {
```

Comment:
Can we pick a better name for these `tracked` variables? I find it conceptually confusing to have a variable named `tracked` that is something different than `@tracked`. Maybe `combinedTrackingTag` or `consumedTrackedPropertiesTag`?

Reply:
Yeahhhh, naming is hard lol, but yeah I'll go with the longer name